data/battle_anim_scripts.s

```diff
@@ -16486,7 +16486,78 @@ Move_RUINATION::
 Move_COLLISION_COURSE::
 Move_ELECTRO_DRIFT::
 Move_SHED_TAIL::
+
 Move_CHILLY_RECEPTION::
+	loadspritegfx ANIM_TAG_CONFETTI
+  loadspritegfx ANIM_TAG_PINK_CLOUD
```

Comment:
Tabs in the complete move anim scripts

Reply:
I'm not sure what this means, exactly, so I'm not sure I've addressed it properly...

Reply:
This is what it looks like in the GitHub editor
![image](https://github.com/rh-hideout/pokeemerald-expansion/assets/17243618/80b345d1-89c4-44fc-9e31-bee3f3d6747d)

Reply:
Also here
![image](https://github.com/rh-hideout/pokeemerald-expansion/assets/17243618/124c3e8c-8cdf-4173-a055-6bb363b84941)

Reply:
I think I got all of these (finally). Sorry about the IDE issues here! Should be better going forward!